### PR TITLE
MS SQL Execution module

### PR DESCRIPTION
**Pull request by pakdel**
A Microsoft SQL Server execution module that resembles the Postgres execution module.
It requires FreeTDS and pymssql Python module. something like the following should be enough:

```
yum install freetds pymssql
```

### CODE REVIEW

**Comment by pakdel** [comment]
@techhat Could you please run PyLint on it again?

**Comment by techhat** [comment]
@pakdel, Jenkins runs again automagically when you add a commit, so it's already started.

**Comment by pakdel** [comment]
@techhat I have no idea why PyLint is giving me `W1699: Incompatible Python 3 code found. Proposed fix:`!
I could compile the file using python 3.4 as following:
```
python3.4 -m py_compile  salt/modules/mssql.py
```

**Comment by techhat** [comment]
@pakdel I don't see what it's talking about either. Maybe it's picking up a weird false positive on that blank line?

@s0undt3ch, any thoughts here?

File: salt/modules/mssql.py [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+        mssql.database: ''
+
+    The default for the port is '1433' and for the database is '' (empty string);
+    in most cases they can be left at the default setting.
+    Options that are directly passed into functions will overwrite options from
+    configs or pillars.
+'''
+
+# Import python libs
+from json import JSONEncoder, loads
```
Comment (rallytime) [inline salt/modules/mssql.py]: `from __future__ import absolute_import` - that should help clean up that pylint error.

**Comment by cachedout** [comment]
+1 This is solid work. Well done.

**Comment by pakdel** [comment]
Thank you @rallytime, it's fixed!

**Comment by rallytime** [comment]
No problem! Thanks for fixing that up. This looks great!

**Pull request merged**